wagtailmenus/tests/test_models.py

```diff
@@ -276,3 +279,22 @@ def test_linkpage_redirects_when_served(self):
             response,
             '/superheroes/marvel-comics/spiderman/?somevar=value'
         )
+
+
+class TestNoAbsoluteUrlsPage(TestCase):
+    fixtures = ['updated_test.json']
```

Comment:
Thanks, this is very thorough! However, recently I've been leaning towards having specific test cases like this create their own one-off pages to test with. Fixtures are well suited for some things (like efficiently populating a page tree with a reliable structure to render menus with), but less useful / clear for more 'unit-test like' things. Here's how I did the tests for the new AbstractLinkPage, if you'd like to see an example: https://github.com/rkhleics/wagtailmenus/blob/master/wagtailmenus/tests/test_models.py#L146